src/Surfnet/ServiceProviderDashboard/Application/Command/Service/EditServiceCommand.php

```diff
@@ -51,7 +50,7 @@ public function __construct(
         private ?string $intakeStatus,
         private ?string $contractSigned,
         private ?string $surfconextRepresentativeApproved,
-        private ?string $privacyQuestionsAnswered,
+        private bool $privacyQuestionsAnswered,
```

Comment:
not nullable anymore?

BTW, I heard a statement, that bool variables "sh|c"ould be named with `is` in front, like $isPrivacyQuestionAnswered.

Butt hat is probably not for now

Reply:
If is is not nullebale, maybe initilize it with false to be sure?

Reply:
I looked it up, and it should always be set. My own testing and the CI functional tests did confirm my assumption. I feel confident to leave it like this.

Reply:
https://3v4l.org/gWhSF